src/rescle.cc

```diff
@@ -438,10 +442,24 @@ bool ResourceUpdater::Load(const WCHAR* filename) {
   EnumResourceNamesW(hModule, RT_VERSION, OnEnumResourceName, reinterpret_cast<LONG_PTR>(this));
   EnumResourceNamesW(hModule, RT_GROUP_ICON, OnEnumResourceName, reinterpret_cast<LONG_PTR>(this));
   EnumResourceNamesW(hModule, RT_ICON, OnEnumResourceName, reinterpret_cast<LONG_PTR>(this));
+  EnumResourceNamesW(hModule, RT_MANIFEST, OnEnumResourceManifest, reinterpret_cast<LONG_PTR>(this));
 
   return true;
 }
 
+bool ResourceUpdater::SetExecutionLevel(const WCHAR* value) {
+  std::wstring ws(value);
+  executionLevel = std::wstring(ws.begin(), ws.end());
```

Comment:
I am keeping them all in wstring so that I can easily manipulate and compare later, and to keep them consistent with the other values stored in ResourceUpdater.

If you would prefer it to be handled a different way, just let me know and I can rewrite it.